src/mpsc/async_impl.rs

```diff
@@ -257,6 +257,81 @@ feature! {
                 .core
                 .try_send(self.inner.slots.as_ref(), val, &self.inner.recycle)
         }
+
+        /// Returns the *total* capacity of the channel for this [`Sender`].
+        /// This includes both occupied and unoccupied entries.
+        ///
+        /// To determine the channel's remaining *unoccupied* capacity, use
+        /// [`remaining`] instead.
+        ///
+        /// # Examples
+        ///
+        /// ```
+        /// use thingbuf::mpsc::channel;
+        ///
+        /// let (tx, _) = channel::<usize>(100);
+        /// assert_eq!(tx.capacity(), 100);
+        /// ```
+        ///
+        /// Even after sending several messages, the capacity remains
+        /// the same:
+        /// ```
+        /// # use thingbuf::mpsc::channel;
+        ///
+        /// let (tx, rx) = channel::<usize>(100);
+        /// *tx.try_send_ref().unwrap() = 1;
+        /// *tx.try_send_ref().unwrap() = 2;
+        /// *tx.try_send_ref().unwrap() = 3;
+        ///
+        /// assert_eq!(tx.capacity(), 100);
+        /// ```
+        ///
+        /// [`remaining`]: Self::remaining
+        #[inline]
+        pub fn capacity(&self ) -> usize {
+            self.inner.core.core.capacity()
+        }
+
+        /// Returns the unoccupied capacity of the channel for this [`Sender`]
+        /// (i.e., how many additional elements can be sent before the channel
+        /// will be full).
+        ///
+        /// This is equivalent to subtracting the channel's [`len`] from its [`capacity`].
+        ///
+        /// [`len`]: Self::len
+        /// [`capacity`]: Self::capacity
```

Comment:
again, maybe worth a 
```suggestion
        /// [`capacity`]: Self::capacity
        #[must_use]
```